external-api/src/interfaces/eStore.interface.ts

```diff
@@ -36,49 +40,54 @@ export interface EstoreSite {
 
 /**
  * Represents a term store within the eStore, identified by a unique string ID.
+ *
  * @interface EstoreTermStore
- * @property {string} id - Unique identifier for the term store.
+ * @property {number} facilityId - Unique identifier for the term store.
  */
 export interface EstoreTermStore {
-  readonly id: string;
+  readonly id: number;
 }
 
 /**
  * Extends EstoreSite to include buyer information, specifically the buyer's name.
+ *
  * @interface EstoreBuyer
  * @extends EstoreSite
  * @property {string} buyerName - Name of the buyer.
  */
-export interface EstoreBuyer extends EstoreSite {
+export interface EstoreBuyer {
   readonly buyerName: string;
 }
 
 /**
  * Represents a deal folder within the eStore, including deal and market information.
+ *
  * @interface EstoreDealFolder
  * @extends EstoreBuyer
  * @property {string} dealIdentifier - Unique identifier for the deal.
  * @property {string} destinationMarket - Destination market for the deal.
  * @property {string} riskMarket - Risk market for the deal.
  */
-export interface EstoreDealFolder extends EstoreBuyer {
+export interface EstoreDealFolder extends EstoreSite, EstoreBuyer {
   readonly dealIdentifier: string;
   readonly destinationMarket: string;
   readonly riskMarket: string;
 }
 
 /**
  * Represents a facility folder within the eStore, identified by a facility identifier.
+ *
  * @interface EstoreFacilityFolder
  * @extends EstoreBuyer
  * @property {string} facilityIdentifier - Unique identifier for the facility.
```

Comment:
is this the correct type?

Reply:
Good spot have updated with an additional `@extends`